frontend/package.json

```diff
@@ -79,7 +79,7 @@
     "cypress": "13.7.2",
     "cypress-vite": "^1.4.0",
     "date-fns": "2.30.0",
-    "date-fns-tz": "^2.0.0",
+    "date-fns-tz": "^3.0.0",
```

Comment:
One braking change is that date-fns v2 is no longer supported but we still have date-fns v2... right above this one